src/sentry/utils/outcomes.py

```diff
@@ -72,11 +72,12 @@ def track_outcome(
     assert isinstance(quantity, int)
 
     outcomes_config = settings.KAFKA_TOPICS[settings.KAFKA_OUTCOMES]
-    billing_config = settings.KAFKA_TOPICS.get(settings.KAFKA_OUTCOMES_BILLING) or outcomes_config
+    billing_config = settings.KAFKA_TOPICS.get(settings.KAFKA_OUTCOMES_BILLING)
+    use_billing = billing_config is not None and outcome.is_billing()
```

Comment:
This is one of the more important changes. Instead of defaulting the `billing_config` to the regular outcomes topic configuration and using the topic name from there, we now check if `KAFKA_TOPICS` has an entry for `"outcomes-billing"` and fall back to using `KAFKA_OUTCOMES` for everything.

Reply:
That's the intent, yes :)